app/src/main/java/com/codingblocks/cbonlineapp/course/CourseViewModel.kt

```diff
@@ -221,4 +235,87 @@ class CourseViewModel(
         }
         return LivePagedListBuilder<String, Course>(dataSourceFactory, config)
     }
+
+    fun checkIfWishlisted(allCourse: List<Course>?){
+        runIO {
+            allCourse?.forEach {courseSingle->
```

Comment:
no need for this

Reply:
Sir wishlist mein jo aap remove karne ko kehe rhe ho, IF we removed that, then how will we know that the course is wishlisted or not??
Because API response mein hume nhi aata ye
Website pe bhi esse hi check kar rhe hai